src/main/java/frc/robot/subsystems/drive/CommandSwerveDrivetrain.java

```diff
@@ -238,11 +257,16 @@ private void controlDrivetrain() {
     }
 
     private Rotation2d calculateDesiredHeading(Pose2d current, Pose2d target) {
-        double robotXAnticipated =
-                current.getX() + (getRobotVelocity.get().getX() * DriveConstants.anticipationTime);
+        Translation2d robotVelocityAdjusted =
+                getRobotVelocity.get().times(DriveConstants.anticipationTime);
+
+        if (Math.abs(robotVelocityAdjusted.getNorm())
```

Comment:
Does getNorm not already produce a non-negative number? (i.e. can we remove the `abs`?)

Reply:
You're right, that is now deleted